.gitignore

```diff
@@ -0,0 +1,4 @@
+*.jpg
```

Comment:
I thought that was standard to not add to the repository

Reply:
No, this was just used as an example, definitely do not use that.

I am not sure what the /Img listing is for either, it will exclude that folder from the repo. If this is intentional that is fine, but I advise you ask about this.